apps/console/src/features/identity-providers/api/identity-provider.ts

```diff
@@ -1285,3 +1286,108 @@ export const getIDPConnectedApps = (idpId: string): Promise<any> => {
             return Promise.reject(error);
         });
 };
+
+/**
+ * Get all sms notification senders with name SMSPublisher.
+ *
+ * @returns  A promise containing the response.
+ */
+export const getSMSNotificationSenders = (): Promise<any> => {
+
+    const requestConfig = {
+        headers: {
+            "Accept": "application/json",
+            "Access-Control-Allow-Origin": store.getState().config.deployment.clientHost,
+            "Content-Type": "application/json"
+        },
+        method: HttpMethods.GET,
+        url: store.getState().config.endpoints.notificationSenders + "/sms"
+    };
+
+    return httpClient(requestConfig)
+        .then((response) => {
+            if (response.status !== 200) {
+                return Promise.reject(
+                    new Error("Failed to get SMS Notification Senders")
+                );
+            }
+            return Promise.resolve(response.data as NotificationSenderSMSInterface[]);
+        }).catch((error) => {
+            return Promise.reject(error);
+        });
+};
+
+/**
+ * Add sms notification senders with name SMSPublisher.
+ *
+ * @returns  A promise containing the response.
+ */
+export const addSMSPublisher = (): Promise<any> => {
```

Comment:
Avoid `any`. Create an interface if neccessary.

Reply:
resolved in https://github.com/wso2/identity-apps/pull/3699/commits/89a5b7ce383f598b61ffaab02e48b0a23c2bf27a